pynn_spinnaker/simulator.py

```diff
@@ -340,6 +351,9 @@ def _load_current_input_verts(self, placements, allocations,
                 # Select placed chip
                 with self.machine_controller(x=vertex_placement[0],
                                              y=vertex_placement[1]):
+                    # Configure watchdog for this vertex
+                    self._configure_software_watchdog()
```

Comment:
DUH! From a placements dictionary, is ``set(itervalues(placements))`` the best way of getting all chips?